os/binfmt/libelf/libelf_init.c

```diff
@@ -95,6 +95,52 @@
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: bin_header_info
+ *
+ * Description:
+ *  Get the binary header info
+ *
+ * Returned Value:
+ *   0 (OK) is returned on success and a negated errno is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ #ifdef CONFIG_ELF_MTD
+ struct bininfo {
+	 int st_size;
+	 int hdr_size;
+ }bininfo;
+ static inline int elf_bin_info(FAR struct elf_loadinfo_s *loadinfo, FAR const char *filename)
```

Comment:
Any reason for inline?

Reply:
Not specific, just to avoid overhead of function call and push/pop variables.